### ISSUE TITLE
Absent corneal reflex

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7465]
**Preferred term label:**
Absent corneal reflex

**Synonyms**
Absent corneal blinking reflex

**Definition (free text, please give PubMed ID)**
Absence of the corneal blinking reflex, which normally induces involuntary blinking of the eyelids following contact of the cornea. The corneal blink reflex is caused by a loop between the trigeminal sensory nerves and the facial motor (VII) nerve innervation of the orbicularis oculi muscles.
PMID:30480980 

Also move to physiology

Comment: The corneal reflex can be tested by touching the cornea lightly with a cotton tip. The normal response to to blink (shut the eyelids) immediately.

To evaluate the corneal reflex, the examiner uses a cotton-tipped applicator, the tip of which has been pulled into a thin strand about 1.3 cm in length. The examiner stabilizes the patient's head by placing a hand on the patient's eyebrow and head. The patient is asked to look to the left side as the cotton tip is brought in from the right side to touch the right cornea gently. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601559 
PMID:14740318  8/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
